document/text/values.rst

```diff
@@ -0,0 +1,260 @@
+     s{:}\Tsign~~\text{0x}~~n{:}\Thexnum &\Rightarrow& s\cdot n & (-2^{N-1} \leq s\cdot n < 2^{N-1}) \\
+   \end{array}
+
+:ref:`Uninterpreted integers <syntax-int>` can be written as either signed or unsigned, and are normalized to unsigned in the abstract syntax.
+
+.. math::
+   \begin{array}{llclll@{\qquad\qquad}l}
+   \production{uninterpreted integers} & \TiX{N} &::=&
+     n{:}\TuX{N} &\Rightarrow& n \\ &&|&
+     i{:}\TsX{N} &\Rightarrow& n & (i = \signed(n)) \\
+   \end{array}
+
+
+.. _text-frac:
+.. _text-hexfrac:
+.. _text-float:
+.. _text-hexfloat:
+.. index:: floating-point number
+   pair: text format; floating-point number
+
+Floating-Point
+~~~~~~~~~~~~~~
+
+:ref:`Floating point <syntax-float>` values can be represented in either decimal or hexadecimal notation.
+The value of a literal must not lie outside the representable range of the corresponding `IEEE 754 <http://ieeexplore.ieee.org/document/4610935/>`_ type
+(that is, a numeric value must not overflow to :math:`\pm\mbox{infinity}`),
+but it may be rounded to the nearest representable value.
+
+.. note::
+   Rounding can be prevented by using hexadecimal notation with no more significant bits than supported by the required type.
+
+Floating-point values may also be written as constants for *infinity* or *canonical NaN* (*not a number*).
+Furthermore, arbitrary NaN values may be expressed by providing an explicit payload value.
+
+.. math::
+   \begin{array}{llclll@{\qquad\qquad}l}
+   \production{decimal floating-point fraction} & \Tfrac &::=&
+     \epsilon &\Rightarrow& 0 \\ &&|&
+     d{:}\Tdigit~q{:}\Tfrac &\Rightarrow& (d+q)/10 \\
+   \production{hexadecimal floating-point fraction} & \Thexfrac &::=&
+     \epsilon &\Rightarrow& 0 \\ &&|&
+     h{:}\Thexdigit~q{:}\Thexfrac &\Rightarrow& (h+q)/16 \\
+   \production{decimal floating-point number} & \Tfloat &::=&
+     s{:}\Tsign~p{:}\Tnum~\text{.}~q{:}\Tfrac
+       &\Rightarrow& s\cdot(p+q) \\ &&|&
+     s{:}\Tsign~p{:}\Tnum~(\text{E}~|~\text{e})~t{:}\Tsign~e{:}\Tnum
+       &\Rightarrow& s\cdot p\cdot 10^{t\cdot e} \\ &&|&
+     s{:}\Tsign~p{:}\Tnum~\text{.}~q{:}\Tfrac~(\text{E}~|~\text{e})~t{:}\Tsign~e{:}\Tnum
+       &\Rightarrow& s\cdot(p+q)\cdot 10^{t\cdot e} \\
+   \production{hexadecimal floating-point number} & \Thexfloat &::=&
+     s{:}\Tsign~\text{0x}~p{:}\Thexnum~\text{.}~q{:}\Thexfrac
+       &\Rightarrow& s\cdot(p+q) \\ &&|&
+     s{:}\Tsign~\text{0x}~p{:}\Thexnum~(\text{P}~|~\text{p})~t{:}\Tsign~e{:}\Tnum
+       &\Rightarrow& s\cdot p\cdot 2^{t\cdot e} \\ &&|&
+     s{:}\Tsign~\text{0x}~p{:}\Thexnum~\text{.}~q{:}\Thexfrac~(\text{P}~|~\text{p})~t{:}\Tsign~e{:}\Tnum
+       &\Rightarrow& s\cdot(p+q)\cdot 2^{t\cdot e} \\
+   \production{floating-point value} & \TfX{N} &::=&
+     z{:}\Tfloat &\Rightarrow& b^\ast & (\ieee_N(z) = b^\ast) \\ &&|&
+     z{:}\Thexfloat &\Rightarrow& b^\ast & (\ieee_N(z) = b^\ast) \\ &&|&
+     s{:}\Tsign~\text{inf} &\Rightarrow& b^\ast & (\ieeeinf_N(s) = b^\ast) \\ &&|&
```

Comment:
Ah, right, forgot to list that as a removal. It seems a bit redundant to have two spellings. Added to PR description.